app/src/fdroid/google-services.json

```diff
@@ -0,0 +1,40 @@
+{
+  "project_info": {
+    "project_number": "296120793014",
+    "firebase_url": "https://pushnotifications-73c5e.firebaseio.com",
+    "project_id": "pushnotifications-73c5e",
+    "storage_bucket": "pushnotifications-73c5e.appspot.com"
+  },
+  "client": [
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:296120793014:android:34d2ba8d355ca9259a7317",
+        "android_client_info": {
+          "package_name": "org.greatfire.wikiunblocked"
+        }
+      },
+      "oauth_client": [
+        {
+          "client_id": "296120793014-8l3acuf7qvvesvqngoc06j2nk2v0bhin.apps.googleusercontent.com",
+          "client_type": 3
+        }
+      ],
+      "api_key": [
+        {
+          "current_key": "placeholder0000000000000000000000000000"
+        }
+      ],
+      "services": {
+        "appinvite_service": {
+          "other_platform_oauth_client": [
+            {
+              "client_id": "296120793014-8l3acuf7qvvesvqngoc06j2nk2v0bhin.apps.googleusercontent.com",
+              "client_type": 3
+            }
+          ]
+        }
+      }
+    }
+  ],
+  "configuration_version": "1"
+}
```

Comment:
I'm guessing that this file would be unneeded if the _firebase_ and _mlkit_ deps are removed from the _fdroid_ flavor.

Reply:
The wikipedia app is already on fdroid, so the greatfire fdroid should just be as close as possible to that.  I vaguely remember something about it having references to proprietary libs, but then they weren't included in the final APK.

Reply:
it looks as if the original wikipedia build.gradle doesn't assign fdroid an applicationIdSuffix or include a unique google-services.json file in the fdroid directory, so it just relies on the default file in the app folder: https://github.com/wikimedia/apps-android-wikipedia/blob/main/app/build.gradle#L132  that file doesn't include our package name though, so i'll need to keep this version of the file since i don't think i can remove the google-services plugin from the shared build.gradle file.

i'm hoping that by not including the mlkit and firebase dependencies in the fdroid build flavor, and by using the stubbed out version of the classes that required them, that the packages will be excluded from the final apk.